services/app-api/src/emailer/templateHelpers.ts

```diff
@@ -100,12 +105,29 @@ const generateReviewerEmails = (
         submission.stateCode !== 'PR' &&
         !includesChipPrograms(submission.programIDs)
     ) {
-        return contractAndRateReviewerEmails
+        return pruneDuplicateEmails(contractAndRateReviewerEmails)
     } else if (includesChipPrograms(submission.programIDs)) {
-        return chipReviewerEmails
+        return pruneDuplicateEmails(chipReviewerEmails)
+    }
+
+    return pruneDuplicateEmails(cmsReviewSharedEmails)
+}
+
+const generateStateReceiverEmails = (
+    pkg: UnlockedHealthPlanFormDataType | LockedHealthPlanFormDataType,
+    user?: UserType
+): string[] => {
+    const stateReceiverEmails: string[] = []
+    //Only add current user email if user is passed in a role is state user
+    if (user?.email && user?.role === 'STATE_USER') {
+        stateReceiverEmails.push(user.email)
```

Comment:
Is this new logic you are adding?  I thought we wanted to always add as a receiver the person triggering the email (the current user) and not restrict this to state users.

Reply:
I scoped this function to only generate list of state emails so if current user was not a state, their email wouldn't get lumped into the list and be sent an email specifically for the states. The check was to make sure if somehow the user passed in was a CMS user it would skip this step.